nl/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
 # the terms of any one of the MPL, the GPL or the LGPL.
 #
 # ***** END LICENSE BLOCK *****
-IncorrectKey_TryAgain? = Master password incorrect. Do you want to try again?
-WantToCaptureForm? = Do you want to remember the values you filled in?
-WantToSavePasswordObscured? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-WantToSavePasswordEncrypted? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-xWantToSavePasswordObscured? = Do you want to remember your user name and password?##Stored values are not password protected.
-xWantToSavePasswordEncrypted? = Do you want to remember your user name and password?##Stored values are password protected.
+IncorrectKey_TryAgain? = Master wachtwoord onjuist. Wilt u het opnieuw proberen?
+WantToCaptureForm? = Wilt u de waardes die u heeft ingevuld onthouden?
+WantToSavePasswordObscured? = Wachtwoord Manager kan deze inloggevens onthouden en automatisch invullen de volgende keer dat u terugkomt op deze website.##Wilt u dat de Wachtwoord Manager deze inloggevens onthoudt?
+WantToSavePasswordEncrypted? = Wachtwoord Manager kan deze inloggevens onthouden en automatisch invullen de volgende keer dat u terugkomt op deze website.##Wilt u dat de Wachtwoord Manager deze inloggevens onthoudt?
```

Comment:
CORRECTION NEEDED, it should be: ''Wachtwoord Manager kan deze inloggegevens onthouden en automatisch invullen de volgende keer dat u terugkomt op deze website.##Wilt u dat Wachtwoord Manager deze inloggegevens onthoudt?''

inloggevens = inloggegevens